Source/WTF/wtf/text/Base64.cpp

```diff
@@ -277,10 +276,9 @@ String base64DecodeToString(StringView input, Base64DecodeMode mode)\
         return String::adopt(WTFMove(*optionalBuffer));
     };
 
-    unsigned length = input.length();
-    if (!length || input.is8Bit())
-        return toString(base64DecodeInternal<LChar, StringImplMalloc>(std::span(input.characters8(), length), mode));
-    return toString(base64DecodeInternal<UChar, StringImplMalloc>(std::span(input.characters16(), length), mode));
+    if (input.isEmpty() || input.is8Bit())
```

Comment:
I think this only needs to check `is8Bit`. I don’t think `isEmpty` leads to additional optimization or is needed for correctness.